common/src/jni/main/cpp/org_conscrypt_NativeCrypto.cpp

```diff
@@ -9173,7 +9180,7 @@ static bool get_ocsp_single_responses(CBS *basic_response, CBS *single_responses
     }
 
     // Extract the list of SingleResponse.
-    return CBS_get_asn1(&response_data, single_responses, CBS_ASN1_SEQUENCE);
+    return CBS_get_asn1(&response_data, single_responses, CBS_ASN1_SEQUENCE) != 0;
```

Comment:
I like it more when we use "== 1" but that might be a holdout when OpenSSL returned a tri-state with 0 meaning not successful and -1 meaning error.